pyiron_atomistics/atomistics/structure/pyscal.py

```diff
@@ -234,13 +223,63 @@ def analyse_voronoi_volume(atoms):
     Args:
         atoms : (pyiron_atomistics.structure.atoms.Atoms): The structure to analyze.
     """
-    s.publication_add(publication())
-    sys = pc.System()
-    sys.read_inputfile(atoms, format="ase")
+    sys = pyiron_to_pyscal_system(atoms)
     sys.find_neighbors(method="voronoi")
     atoms = sys.atoms
     return np.array([atom.volume for atom in atoms])
 
+def pyiron_to_pyscal_system(atoms):
+    """
+    Converts atoms to ase atoms and than to a pyscal system.
+    Also adds the pyscal publication.
+
+    Args:
+        atoms (pyiron atoms): Structure to convert.
+
+    Returns:
+        Pyscal system: See the pyscal documentation.
+    """
+    s.publication_add(publication())
+    sys = pc.System()
+    sys.read_inputfile(
+        pyiron_atomistics.atomistics.structure.atoms.pyiron_to_ase(atoms),
+        format="ase",
+        )
+    return sys
+
+def analyse_find_solids(atoms, neighbor_method="cutoff",
```

Comment:
This returns the number of *atoms* that are inside a solid phase?  If so it'd cool to also return an index array, that shows which are solid and which are liquid, maybe behind a switch, like the `pyscal_adaptive_cna` method.

Reply:
This only returns the number by default, but when using return_system=True it returns the complete pyscal system object, which allows to get the indices. I thought this is the most simple in the default case and the most flexible approach when using the return_system option.
Alternatively I could change the default to return an array with the indices of all atoms that are identified as solid, so that the len(returned_array) gives the number of solids I guess. But I do not know if the order of the atoms stays the same when converting between pyiron ase and pyscal system, so I am not sure if indices will always be correct

Reply:
Ah, I had simply missed that while skimming.  Since anything might happen while changing from pyiron to pyscal returning the full system is ok, I suppose.